inst/include/Rcpp/exceptions.h

```diff
@@ -127,17 +127,17 @@ namespace Rcpp{
 } // namespace Rcpp
 
 inline SEXP get_last_call(){
-    SEXP sys_calls_symbol = Rf_install( "sys.calls" ) ;
-    Rcpp::Shield<SEXP> sys_calls_expr( Rf_lang1(sys_calls_symbol) );
+    SEXP sys_calls_symbol = Rf_install( "sys.call" ) ;
+
+    // -9 Skips the wrapped tryCatch from Rcpp_eval
+    Rcpp::Shield<SEXP> sys_calls_expr( Rf_lang2(sys_calls_symbol, Rf_ScalarInteger(-9)) );
```

Comment:
The tricky part of looking at `sys.calls()` and related functions is to ignore intervening frames. `rlang:::trail_make()` is one way of getting the vector of relevant parent frames.

I think if you run `rlang::call_stack()` instead of `sys.calls()` here, you should get only a handful of frames instead of 14.